cypress/integration/landing.spec.js

```diff
@@ -0,0 +1,33 @@
+describe('Landing page', () => {
+  it('redirects to welcome page', () => {
+    cy.visit('/app')
+
+    cy.url().should('include', '/app/welcome')
+  })
+
+  it('features discoverability cookies', () => {
+    cy.visit('https://safe-team.dev.gnosisdev.com/app')
+
+    const cookieBannerId = '[data-testid="cookies-banner-form"]'
+
+    cy.get(cookieBannerId).contains('We use cookies to provide you with the best experience')
```

Comment:
The texts can be extracted into variables to avoid duplication.

Reply:
you can add https://testing-library.com/docs/cypress-testing-library/intro/ and use cy.getByTestId("cookies-banner-form")

As well you are going to receive a lot of nice selectors